src/content/docs/extensions/algo/spanningforest.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: Spanning Forest
+---
+
+The Spanning Forest algorithm finds a minimal set of edges that connect all nodes within each connected component
```

Comment:
For people who are new to this, they're going to find the name [as per wikipedia](https://en.wikipedia.org/wiki/Kruskal%27s_algorithm) specified as "Minimum spanning forest". Could we work the word "minimum" somewhere in here? That would also lead more google/AI searches to this.